.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: c
+dist: trusty
+
+matrix:
+  include:
+  - env: DISTRO=ubuntu IMAGE=ubuntu:xenial
```

Comment:
What other distros and combinations are available? I'd like to have at least one Fedora here, possibly Debian Stretch, and you also mentioned building with both GCC and Clang which I would very much like to have in this matrix.